devdoc/CodeReviews.md

```diff
@@ -0,0 +1,88 @@
+discuss: | Reviewer has thought of a potential problem, but not sure if it applies | Reviewer has a concern it may be an issue, and wants to check the developer has thought about and addressed the issue | Discussion - either in the PR or offline.
+||
+style: |  Reviewer points out non-conforming code style | Makes the code hard to read | Developer to fix
+indent: | A fairly obvious indentation issue | Makes the code hard to read | Developer to fix.
+format: | Any other unusual formatting | Makes the code hard to read | Developer to fix.
+typo: | Minor typing error | Makes something (code/message/comment) harder to read | Developer to fix.
+minor: | A minor issue that could be improved. | Education (the suggestion is better for a particular reason), or something simple to clean up at the same time as other changes | Developer recommended to fix, but unlikely to stop a merge
+picky: | A very minor issue that could be improved, but is barely worth commenting on | Education, or something to clean up at the same time as other changes | Developer discretion to fix, wouldn't stop a merge
+||
+future: | An additional feature or functionality that fits in but should be done as a separate PR. | Ensure that missing functionality is tracked, but PRs are not held up by additional requirements. | Contributor to create Jira (unless trivial) and number noted in response.
+question: | Review has a question that they are not sure of the answer to | Reviewer would like clarification to help understand the code or design.  The answer may lead to further comments. | An answer to the question.
+note: | Reviewer wants to pass on some information to the contributor which they may not know | Passing on knowledge/background | contributor should consider the note, but no change expected/required
+personal: | Reviewer has an observation based on personal experience | Reviewer has comments that would improve the code, but not part of the style guide or required.  E.g. patterns for guard conditions  | Reflect on the suggestion, but no change expected.
+documentation: | This change may have an impact on documentation | Make sure changes can be used | Contributor to create Jira describing the impact created, and number noted in response.
+
+The comments should always be constructive.  The reviewer should have a reason for each of them, and be able to articulate the reason in the comment or when asked.  "I wouldn't have done it like that" is not a good enough on its own!
+
+Similarly there is a difference in opinion within the team on some style issues - e.g. standard libraries or jlib, inline or out of line functions, nested or non-nested classes.  Reviews should try and avoid commenting on these unless there is a clear reason why they are significant (functionality, efficiency, compile time) and if so spell it out.  Code reviewers should discuss any style issues that they consider should be universally adopted that are not in the style guide.
```

Comment:
Perhaps a link to the style guide would be handy (and perhaps we should consider updating it)